### PR TITLE
Update README.md with v1 announcement.

### PR DESCRIPTION
**Pull request by mohsinenar**

### CODE REVIEW

**Review by elyousfi5 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -3,12 +3,20 @@
 [![Ostorlab blog](https://img.shields.io/badge/blog-ostorlab%20news-red)](https://blog.ostorlab.co/)
 [![Twitter Follow](https://img.shields.io/twitter/follow/ostorlabsec.svg?style=social)](https://twitter.com/ostorlabsec)
 
+> [!IMPORTANT]
+> OXO (previously known as Ostorlab) has reached v1.0.0, [Release announcement](https://oxo.ostorlab.co/docs/whats_new).
+
```
[the remark this answers is not in the document]
Comment (3asm) [reply README.md:8]: Command lineks can be nicer update it.

[the remark this answers is not in the document]
Comment (3asm) [reply README.md:8]: Example has a dedicated section, reference it and remove them from here.

[the remark this answers is not in the document]
Comment (3asm) [reply README.md:8]: Remove the pitch section.

**Review by 3asm — COMMENTED** [review]
Not your best work @mohsinenar I am used to excellent'er work from you.

File: README.md [hunk 2]
```diff
@@ -72,7 +78,7 @@ oxo vulnz describe --vuln-id <vuln-id>
 ```
 
 # Docker Image 
-To run `oxo` in a container, you may use the publically available image and run the following command:  
+To run `oxo` in a container, you may use the publicly available image and run the following command:
```
Comment (3asm) [inline README.md:81]: You forgot to update the screenshots.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**